docs/content/comparisons/druid-vs-storage-formats.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: doc_page
+---
+
+Druid vs Storage Formats (Parquet/Kudu)
+=======================================
+
+The biggest difference between Druid and existing storage formats is that Druid includes an execution engine that can run 
+queries, and a time-optimized data management system for advanced data retention and distribution,. 
+The Druid segment is a custom column format designed for fast aggregates and filters. Below we compare Druid's segment format to 
+other existing formats. 
+
+## Druid vs Parquet
+
+Druid's storage format is highly optimized for linear scans. Although Druid has support for nested data, Parquet's storage format is much 
+more hierachical, and is more designed for binary chunking. In theory, this should lead to faster scans in Druid.
+
+## Druid vs Kudu
+
+Kudu's storage format enables single row updates, whereas updates to existing Druid segments requires recreating the segment, hence,  
+the update process is slower in Druid. Druid summarizes/rollups up data at ingestion time, which in practice reduces the raw data that needs to be 
```

Comment:
Instead of
> the update process is slower in Druid
We should say
> the process for updating old values is slower in Druid.  The requirements in Kudu for maintaining extra head space to store updates as well as organizing data by id instead of time has the potential to introduce some extra latency and accessing of data that is not need to answer a query at query time.